src/components/bookCategories.js

```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>Cooming soon</h1>
```

Comment:
- [ ] Great job on making the set up but I have a small note I think you forget to add a button here for check status as required in the requirements so i recommend to add it here 💯💯🔥😊
![image](https://user-images.githubusercontent.com/58553711/213895418-1bcc1706-0dd2-4f64-88ad-1e752a8fe0c5.png)

- Your project
![image](https://user-images.githubusercontent.com/58553711/213895432-e80f8e71-8cfa-455d-a624-e381bdb33d64.png)